pkg/solver/singlehost.go

```diff
@@ -206,7 +206,7 @@ func (c *CheRoutingSolver) getGatewayConfigMaps(cheManager *dwoche.CheManager, w
 			i := int32(e.TargetPort)
 
 			name := ""
-			if e.Attributes.GetString(uniqueEndpointAttributeName, nil) == "true" {
+			if e.Attributes.GetBoolean(uniqueEndpointAttributeName, nil) {
```

Comment:
I guess it does work with property being written as string? https://github.com/che-incubator/devworkspace-che-operator/blob/935e25160bff568c498a575fa01d4679b200bce6/samples/flattened_theia-nodejs.yaml#L61

Reply:
Yes, the impl does account for strings.